newIDE/app/src/GameDashboard/LeaderboardAdmin/LeaderboardSortOptionsDialog.js

```diff
@@ -0,0 +1,220 @@
+        );
+        return;
+      }
+      if (extremeAllowedScoreValue < extremeAllowedScoreMin) {
+        setExtremeAllowedScoreError(
+          i18n._(
+            t`Extreme score must be equal or higher than ${extremeAllowedScoreMin}.`
+          )
+        );
+        return;
+      }
+    }
+    if (!isWholeNumber(extremeAllowedScoreValue)) {
+      setExtremeAllowedScoreError(
+        i18n._(
+          t`The extreme score must be a whole number between ${extremeAllowedScoreMin} and ${extremeAllowedScoreMax}.`
+        )
+      );
+      return;
+    }
+    setIsLoading(true);
+    const sortOrderSettings = {
+      sort: sortOrder,
+      extremeAllowedScore: displayExtremeAllowedScoreInput
+        ? extremeAllowedScoreValue
+        : null,
+    };
+    await onSave(sortOrderSettings);
+  };
+
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <Dialog
+          open={open}
+          maxWidth="sm"
+          onRequestClose={() => {
+            if (!isLoading) onClose();
+          }}
+          secondaryActions={[
+            <HelpButton
+              key="help"
+              helpPagePath="/interface/games-dashboard/leaderboard-administration"
+              anchor="change_the_appearance_of_a_leaderboard"
+            />,
+          ]}
+          actions={[
+            <FlatButton
+              label={<Trans>Cancel</Trans>}
+              disabled={isLoading}
+              onClick={onClose}
+              key={'cancel'}
+            />,
+            <RaisedButton
+              primary
+              label={<Trans>Save</Trans>}
+              disabled={isLoading}
+              onClick={() => onSaveSettings(i18n)}
+              key={'save'}
+            />,
+          ]}
+          onApply={() => {
+            onSaveSettings(i18n);
+          }}
+        >
+          <Text size="title">
+            <Trans>Sort order</Trans>
+          </Text>
+          <ColumnStackLayout>
+            <SelectField
+              fullWidth
+              value={sortOrder}
+              floatingLabelText={<Trans>Scores sort order</Trans>}
+              onChange={(e, i, newValue) => {
+                // $FlowIgnore
+                setSortOrder(newValue);
+              }}
+            >
+              <SelectOption
+                key={'ASC'}
+                value={'ASC'}
+                primaryText={t`Lower is better`}
+              />
+              <SelectOption
+                key={'DESC'}
+                value={'DESC'}
+                primaryText={t`Higher is better`}
+              />
+            </SelectField>
+            <Checkbox
+              label={
+                <>
+                  <Line noMargin>
+                    <Trans>Limit scores</Trans>
+                  </Line>
+                  <FormHelperText style={{ display: 'inline' }}>
+                    <MarkdownText
+                      source={i18n._(
+                        `Any submitted score that is ${
```

Comment:
Translation will be funky: `Un score soumis qui est lower que 5 ne sera pas sauvegardé`.
Prefer having the choice between 2 whole translations